tests/integration/courseSlots.test.js

```diff
@@ -247,6 +247,18 @@ describe('COURSE SLOTS ROUTES - PUT /courseslots/{_id}', () => {
       expect(response.statusCode).toBe(403);
     });
 
+    it('should return 403 if course slot has attendances', async () => {
```

Comment:
tu pourrais preciser l'action ici. Genre : `it('should return 403 as trying to remove date and course slot has attendances'`

Reply:
done